README.Rmd

```diff
@@ -70,3 +71,31 @@ qgis_run_algorithm(
 unlink("test-file.json")
 ```
 
+The following examples demonstrates the [buffer](https://docs.qgis.org/testing/en/docs/user_manual/processing_algs/qgis/vectorgeometry.html#buffer) algorithm in action.
+
+```{r buffer}
+input_file <- sf::read_sf(system.file("shape/nc.shp", package = "sf"))
+output_file <- file.path(tempdir(), "nc_buffered.gpkg")
+qgis_run_algorithm(
+  "native:buffer",
+  INPUT = input_file,
+  DISTANCE = 1,
```

Comment:
One question: why does the value of '1' lead to what seems like a 1 degree buffer? Any ideas @paleolimbot ? I was expecting that to be a 1m buffer... May thing though, it works :tada: